pertzlab/arcos-gui
# enable loading data from labelproperties

- add python 3.11 to testing
- update arcos4py dependency
- enable data from layer.properties
- set layer output dimension
- toggle which layers get created
- slightly rework callback system to imporve readablity
- add functions to interact with the plugin from e.g. jupyter (open, load_data, run arcos etc)
- fix bug in threading logic
- update tests, add new tests
- remove timestamp widget (now separate plugin)
- updates to type annotations
- missing columns are now indicated by None and not "None"
- cleaned up thread creation and uses now naparis helper thingies to create threads
- support for exporting all settings, now as yaml file
- support for importing settings and parameters from yaml file
- batch processing of entire folders
- updated get_last_instance class method to also check if the last instance is still valid
- various other small changes
